.eslintrc.js

```diff
@@ -69,7 +69,7 @@ module.exports = {
                 "allowKeywords": true
             }
         ],
-        "eol-last": "off",
+        "eol-last": "error",
```

Comment:
github complains about lack of trailing EOLs, so lets make github happy - https://eslint.org/docs/rules/eol-last